Pod/Classes/Optional Data Sources/VOKCollectionDataSource.m

```diff
@@ -270,19 +275,25 @@ - (void)controllerDidChangeContent:(NSFetchedResultsController *)controller
                             if (fetchLimit > 0) {
                                 if (indexPath.row < fetchLimit && newIndexPath.row < fetchLimit) {
                                     // Before and after are both in range
-                                    [self.collectionView moveItemAtIndexPath:indexPath toIndexPath:newIndexPath];
+                                    if (newIndexPath) {
+                                        [self.collectionView moveItemAtIndexPath:indexPath toIndexPath:newIndexPath];
+                                    }
```

Comment:
Would it make sense to test `newIndexPath` at the top of the `NSFetchedResultsChangeMove` case?  It's being used in various tests outside of these checks as it is.